next.config.js

```diff
@@ -1,115 +1,124 @@
 const { withPlausibleProxy } = require('next-plausible')
+const withPWA = require('next-pwa')
+
 
 const isProd = process.env.NODE_ENV === 'production'
 const corsHeaders = [
-  {
-    key: 'Access-Control-Allow-Origin',
-    value: '*'
-  },
-  {
-    key: 'Access-Control-Allow-Methods',
-    value: 'GET, HEAD, OPTIONS'
-  }
+    {
+        key: 'Access-Control-Allow-Origin',
+        value: '*'
+    },
+    {
+        key: 'Access-Control-Allow-Methods',
+        value: 'GET, HEAD, OPTIONS'
+    }
```

Comment:
Please undo code style changes like this.

I assume it was just auto-formatting but changes like these make it hard to see what was actually changed and what the actually required changes are.